src/hub.html

```diff
@@ -207,6 +207,8 @@
             spawn-controller="radius: 4;"
             wasd-to-analog2d
             character-controller="pivot: #player-camera"
+            hand-poses="mixer: .RootScene"
+            hand-poses-controller
```

Comment:
I'd move both of these onto RootScene so they get recreated any time an avatar is switched.